geospaas/base_viewer/templates/base_viewer/elements.html

```diff
@@ -15,7 +15,7 @@
         <td class="place_ds">
             <div class="geometry_ref" id="{{ ds.geographic_location.pk }}">
             {% for url in ds.dataseturi_set.all %}
-            {{ url.uri|urlizetrunc:64}} <br>
+            {{ url.uri|urlizetrunc:264}} <br>
```

Comment:
Why not just use the `urlize` filter? That way you wouldn't have to worry about the length.

Reply:
I was afraid of appearance of the web-page that has many lines of url for only a single file. And this is for every file in the result section!

Reply:
OK, but then changing this number won't solve the problem of incomplete urlization, will it? It only affects the length of the displayed URL.

Reply:
Yeah, you are right! It can also solve the problem if we 100% sure that the urls are shorter than the 264 for any case (in the webpage and this usage of template).

Thus, it is better to keep the safe approach (your choice).

Reply:
done-check

Reply:
You are right, it is probably better to keep the urlizetrunc filter, but you also need to find out why the URLs are partially "urlized"